### DISCUSSION TITLE
Total Time Disabled on GUI

### DISCUSSION THREAD

**Discussion by dyk74** [gpx-animator/gpx-animator#668] [Q&A] [answered]
Hello,
I've just installed GPX-animator and I'testing it... I don't know how to modify it:
the "Total Time" is always disabled, I've tried to change "Speedup"... add speedup, less speedup, remove it but "Total Time" is always disabled.
How can I enable it and choose exactly the Time Lenght?
Thank you

BYE BYE

**Comment by McPringle** [comment, marked as answer]
The values for "speedup" and "total time" are mutually exclusive. Either you enter a value for "speedup" and GPX Animator calculates the duration itself or you enter the desired duration as "total time".

This means: Delete the value of "speedup" and then you can enter a "total time" and vice versa.

**Comment by dyk74** [comment]
Sorry... I'm "stupid" :D

**Reply by McPringle** [reply]
Let's call the UI "stupid". Hopefully I'll have some time to rework the UI this year.
